### PR TITLE
RF/2 Refactor: Begin to extract code chunks from large clusternetwork.js file

### PR DESCRIPTION
**Pull request by daniel-ji**
A couple questions with this PR: 

- Should newly created `tables.js` code and `utils.js` code be merged with `helpers.js` or `misc.js`? Or should we wait to combine since our primary goal is to split things into smaller, modular files and possibly later better name these smaller files to be more specific? 
- When refactoring hivtrace-viz, should we also prettify / format the html files? (Currently they cannot be properly syntax parsed by VSCode)
- Also see question below in `hivtrace.js`. Thank you!

Before vs after formatting html files: (this is from `priority_set.html`):

image

image

### CODE REVIEW

File: src/hivtrace.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { hivtrace_cluster_graph_summary } from "./hivtrace_cluster_graph_summary
 import { histogram, histogramDistances } from "./histogram.js";
 import { scatterPlot } from "./scatterplot.js";
 
+// QUESTION: should these be exported? (seems like they are not being used - misc, helpers, colorPicker)
```
Comment (spond) [inline src/hivtrace.js:7]: Yes, keep them exported for now. They may be used in other contexts. I know that colorPicker is used by @stevenweaver

**Comment by spond** [comment]
Dear @daniel-ji,

1. I would keep the individual `.js` files separate for now. Let's figure out what to do with them a little bit later.
2. I would definitely prettify/forma the `.html` files
3. See my response in the comment.

Best,
Sergei

File: src/hivtrace.js (hunk 1 above)
Comment (stevenweaver) [reply src/hivtrace.js:7]: Yes, I definitely use them.

**Review by spond — APPROVED** [review]

File: src/hivtrace.js (hunk 1 above)
Comment (daniel-ji) [reply src/hivtrace.js:7]: Will do, thank you!

**Pull request merged**